index.ios.js

```diff
@@ -1,13 +1 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- */
-
-import App from './lib/app';
-import { AppRegistry } from 'react-native';
-
-class Root extends App {
-
-}
-
-AppRegistry.registerComponent('Magnet', () => Root);
+require('./index.android.js');
```

Comment:
This seems a bit hacky. I'd rather abstract an `<App>` component and have that used by both `index.android.js` and `index.ios.js`.